app/views/ui-components/v1/cards/_reference_plan_details_shell.html.slim

```diff
@@ -7,29 +7,38 @@
 
 
 .panel.panel-default
-  .panel-body    
-    .col-xs-8
-        h5.heading-text
-          strong#referencePlanTitle
-    .col-xs-4
-      .row#changeReferencePlan.hidden 
-        a onclick="showPlanSelection()" Change reference Plan?      
-    table.table
-      tbody
-        tr
-          th
-            strong Type
-          th
-            strong Carrier
-          th
-            strong Metal Level
-          th
-            strong Network
-        tr
-          td.text-capitalize#rpType
-          td.text-capitalize#rpCarrier
-          td.text-capitalize#rpMetalLevel
-          td.text-capitalize#rpNetwork
+  .panel-body
+    .row
+    .col-md-12.pl-0
+      span.pvp-plan-indicator.mr-1.hidden#rpPvpIndicator
+        i.fas.fa-lg.fa-square.mr-1 data-toggle="tooltip" data-placement="top" data-container="body" title="#{l10n('insured.plan_shoppings.pvp_title_info')}"
```

Comment:
just to confirm - this pvp indicator is not behind a feature flag b/c of initial hidden state - in the code above, this will be selectively unhidden based on feature flag (and pvp-ness of plan)?